src/routes/bitbucket/proxy/index.ts

```diff
@@ -196,15 +196,19 @@ export function proxyRoutes(runner: Runner, client: BitbucketClient) {
 
       const priority = await client.bitbucket.getPullRequestPriority(commit);
 
-      if (priority === 'HIGH' && request) {
-        await request.incrementPriority();
-        Logger.info('Pull request priority increased', {
-          namespace: 'routes:bitbucket:proxy:land',
-          pullRequestId: prId,
-          landRequest,
-        });
-      }
+      if (request) {
+        if (priority === 'HIGH') {
+          await request.incrementPriority();
+          Logger.info('Pull request priority increased', {
+            namespace: 'routes:bitbucket:proxy:land',
+            pullRequestId: prId,
+            landRequest,
+          });
+        }
 
+        const impact = await client.bitbucket.getPRImpact(commit);
```

Comment:
Should this be behind feature toggle? There will be an extra API calls for the landkid instances which don't use speculation engine.

However, I don't see any harm. The impact will be set to 0 if the build statuses are not available.